src/Castle.Core/DynamicProxy/Generators/Emitters/ArgumentsUtil.cs

```diff
@@ -20,7 +20,7 @@ namespace Castle.DynamicProxy.Generators.Emitters
 
 	using Castle.DynamicProxy.Generators.Emitters.SimpleAST;
 
-	internal abstract class ArgumentsUtil
+	internal static class ArgumentsUtil
```

Comment:
Would be good to at least get these `static` changes in. Shows how long ago these classes were first written to use `abstract` to prevent instantiation of the class.